python/lsst/ts/mtdome/mock_llc/mock_motion/base_llc_motion.py

```diff
@@ -48,6 +48,8 @@ def __init__(
         # This is not a constant but can be configured by the MTDome CSC, which
         # is why it is a parameter.
         self._max_speed = max_speed
+        # Keep track of the motion state where we came from.
```

Comment:
It isn't necessarily the most recently commanded motion state, since this may be updated internally. So it is the last known motion state or simply the current motion state. I have updated the comment.